### ISSUE TITLE
Restructuring CSS files

### ISSUE THREAD

**Issue by aldnav** [Danson0406/Testing#2]
1. Create a `styles.css` file
2. Copy `page1.css` file to `styles.css`
3. Load `styles.css` to `JanFebMar.html`. Replacing page1.css with styles.css

```diff
<title>Kalendaryo 2023</title>
++<link rel="stylesheet" href="styles.css">
--<link rel="stylesheet" href="page1.css">
``` 

4. Edit `AprMayJun.html`. Add styles.css before page2.css

```diff
<title>Kalendaryo 2023</title>
++<link rel="stylesheet" href="styles.css">
<link rel="stylesheet" href="page2.css">
```

5. Edit `page2.css`

```css

body {
    background-image: url('background2.jpg');
}

#table2 {
    margin-top: -312px;
}

#page1 {
    margin-left: 30px;
}

#page {
    font-size: 45px;
    margin-left: 30px;
}
#page3 {
    margin-top: -320px;
    margin-left: 30px;
}
div.icon2 {
    margin-left: 555px;
    margin-top: -60px;
}
```

Then do this for other pages (`JulAugSept.html`, `OctNovDec.html`) with `page3.css` and `page4.css` containing only what's unique / different from `styles.css`.

That's it.

Basically what we did is define one base style. The other styles can then just be customizations that will override or add-on the base style.

> It is a good idea to have all of the common styling first in the stylesheet. This means all of the styles which will generally apply unless you do something special with that element.
> - Organizing your CSS, MDN

Suggested readings:

1. https://www.geeksforgeeks.org/how-css-style-overriding-works/
2. https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Organizing
